detector/other/FieldMapBrBz.cpp

```diff
@@ -150,44 +157,156 @@ void FieldMapBrBz::fillFieldMapFromTree(const std::string& filename, const std::
 
   if( strs.size() != 5 ) {
     std::stringstream error;
-    error << "FieldMap[ERROR]: the treeDescription " << treeString
+    error << "FieldMapBrBz[ERROR]: the treeDescription " << treeString
 	  << " is not complete. For example 'fieldmap:rho:z:Brho:Bz'";
     throw std::runtime_error( error.str() );
   }
 
+  //Getting the tree name and branch variables names
```

Comment:
Why is this needed? So that you can now use arbitrary order in the definition of the ntuple?
It was possible to define any field name before, as long as they had the same meaning. Now one can only use the same words except upper or lower case, can't one?

Before I could have a tree like
`myNewFieldMap:Radius:AlongTheDetector:RadialField:AlongTheDetectorField`
Now I can't.